mitodl/micromasters
# Added redux, react-router, neat, google analytics

#### What are the relevant tickets?

Fixes #12 
Fixes #45

#### What's this PR do?

Adds some front-end packages, including redux, react-router, google analytics, bourbon neat.  

#### Where should the reviewer start?

package.json

#### How should this be manually tested?

run docker-compose up, there should be no console errors. Visit /dashboard, and you should see a checkbox, which when clicked, shows redux actions in the js console.

#### Any background context you want to provide?

Setting up the infrastructure for the react app, based on what we learned in Teachers' Portal.

#### Screenshots (if appropriate)

N/A

#### What GIF best describes this PR or how it makes you feel?

![](http://boweryboyshistory.com/wp-content/uploads/2014/10/119.jpg)
